crowbar_framework/app/controllers/application_controller.rb

```diff
@@ -215,4 +216,17 @@ def upgrade
       end
     end
   end
+
+  def check_eol
+    branding = YAML.load_file(Rails.root.join("config", "branding.yml"))
+    return if branding["eol_date"].blank?
+    return if Time.parse(branding["eol_date"]).future?
+
+    flash[:alert] = I18n.t(
+      "dashboard.eol",
+      product: branding["page_title"],
```

Comment:
Ok, removed the date from the branding. Now this PR is very generic and we can activate it from the package.